helpers/nodejs

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-n_version=8.0.2
+n_version=8.2.0
```

Comment:
Eeeeh what about the checsum tho :thinking:

Reply:
Uuuuh but itsn't it at line 19 ?

```bash
   echo "SOURCE_URL=https://github.com/tj/n/archive/v${n_version}.tar.gz
SOURCE_SUM=1cdc34d3a53a13a23675797dd775d562e33e64877e367df9d1afe863719de973" >"$YNH_APP_BASEDIR/conf/n.src"
```

Reply:
So... the updater was calculating the checksum, but not actually putting it in the helper...

Reply:
Cheers !